README.rst

```diff
@@ -2,57 +2,93 @@ pydoctor
 ========
 
 .. image:: https://travis-ci.org/twisted/pydoctor.svg?branch=tox-travis-2
-    :target: https://travis-ci.org/twisted/pydoctor
+  :target: https://travis-ci.org/twisted/pydoctor
 
 .. image:: https://codecov.io/gh/twisted/pydoctor/branch/master/graph/badge.svg
   :target: https://codecov.io/gh/twisted/pydoctor
 
-This is 'pydoctor', an API documentation generator that works by
+This is *pydoctor*, an API documentation generator that works by
 static analysis.
 
 It was written primarily to replace epydoc for the purposes of the
-Twisted project as epydoc has difficulties with zope.interface.  If it
-happens to work for your code too, that's a nice bonus :)
+Twisted project as epydoc has difficulties with zope.interface.
+If you are looking for a successor to epydoc after moving to Python 3,
+pydoctor might be the right tool for your project as well.
 
 pydoctor puts a fair bit of effort into resolving imports and
 computing inheritance hierarchies and, as it aims at documenting
 Twisted, knows about zope.interface's declaration API and can present
 information about which classes implement which interface, and vice
 versa.
 
-The default HTML generator requires Twisted.
+.. contents:: Contents:
 
-There are some more notes in the doc/ subdirectory.
 
+Usage
+-----
 
-Tox development environment
----------------------------
+You can run pydoctor on your project like this::
 
-Since Python 3 is not yet supported, you the case in which your default
-tox runs with Python 3, call the tox as::
+    $ pydoctor --make-html --html-output=docs/api --add-package=src/mylib
 
-    python2 -m tox -e pyflakes
+You can see the full list of options using ``pydoctor --help``.
 
+Markup
+------
+
+pydoctor currently supports the following markup languages in docstrings:
+
+    `epytext`__ (default)
```

Comment:
The level of indentation here is putting this in a blockquote which doesn't seem right.

Reply:
You're right. It looked fine when I converted it to HTML using `rst2html5.py`, so I didn't notice the superfluous blockquote. I'll dedent it.